rocAL/include/loaders/image_source_evaluator.h

```diff
@@ -41,10 +41,14 @@ enum class MaxSizeEvaluationPolicy {
 class ImageSourceEvaluator {
    public:
     ImageSourceEvaluatorStatus create(ReaderConfig reader_cfg, DecoderConfig decoder_cfg);
+    ImageSourceEvaluatorStatus create_numpy_reader(ReaderConfig reader_cfg);
```

Comment:
Please add a new class for NumpySourceEvaluator derived from ImageSourceEvaluator. Clubbing it with ImageSourceEvaluator is very confusing